src/infrastructure/validation/logger/FileLogger.java

```diff
@@ -65,11 +65,11 @@ protected FileLogger(List<LogLevel> enabledLogLevelsList, boolean enableTestMode
 		
 		try {
 			String home = System.getProperty("user.home");
-			logFilePath = home+"/.config/";
+			logFilePath = home+"/.config/KoraKaagaz";
 		} catch (SecurityException se) {
 			// in case a security manager is present, it's checkRead method can deny read access to the file
 			// if it occurs, logFilePath reverts to the current directory where it is run
-			logFilePath = "./.config/";
+			logFilePath = "./.config/KoraKaagaz";
 		}
```

Comment:
Do you want a folder called .config inside present directory, do you need to escape the '.' character? (Also might create problems with being inside a package) logs are referred by all modules, would be better to put it as a folder called 'logs' outside src like you have done for resources

Reply:
this is the last resort location. In case the runtime cant get access to the location `$HOME/.config/KoraKaagaz` , this location is used.
I will change it to ./logs/ or ./config/logs/

Reply:
changing this to `logs/`
In case the primary location is somehow inaccessible, this folder will be created (hopefully) wherever the program was run from